src/reference/config/telemetry/logs.md

```diff
@@ -0,0 +1,95 @@
+---
+shortTitle: Logs
+description: Zilla runtime telemetry logs
+category:
+  - Telemetry
+tag:
+  - Logs
+---
+
+# Telemetry Logs
+
+Named events logged by Zilla.
+
+:::: note Properties
+
+- [Catalog Errors](#catalog-errors)
+- [Filesystem Errors](#filesystem-errors)
+- [Guard Errors](#guard-errors)
+- [HTTP Access](#http-access)
+- [Kafka Errors](#kafka-errors)
+- [Model Errors](#model-errors)
+- [TCP Errors](#tcp-errors)
+- [TLS Errors](#tls-errors)
```

Comment:
In a reference section, it seems more appropriate to group the events generated and owned by a component directly with the reference for that component, rather than separated out into it's own listing and then having to reclassify them again by component and component type.

For example, we have multiple components of type `filesystem`, such as a binding, a vault and a catalog. So it's not clear from this which component is meant because the event is being summarized out of context from the owning component.

If we still need a list, then they could be each be listed and hyperlinked from here.

Reply:
Perhaps it might be more useful to use the `stdout` exporter to illustrate how each event is formatted to `stdout`, providing a unified listing of all the logged events.

Reply:
documenting the decision here:

We will list all events using a more verbose event name based on the full event ID. We will list all of them on this event page and link to them from their individual component pages.